config/jobs/kubernetes-sigs/cluster-api-operator/cluster-api-operator-presubmits-main.yaml

```diff
@@ -163,13 +163,6 @@ presubmits:
             secretKeyRef:
               name: cluster-lifecycle-github-token
               key: cluster-lifecycle-github-token
-        resources:
-          requests:
-            cpu: "1"
-            memory: "2Gi"
-          limits:
-            cpu: "1"
-            memory: "2Gi"
```

Comment:
I think it's worth keeping these and increasing the capacity values to match the other jobs. They will be needed eventually after the secrets are migrated to EKS.

Reply:
Makes sense, done